### PR TITLE
allow-common-devel.inc: Add Arduino

### PR DESCRIPTION
**Pull request by marek22k**
Fixes developing Arduino on VSCodium.

Closes https://github.com/netblue30/firejail/issues/5990

Arduino IDE: https://github.com/arduino/arduino-ide
PlatformIO: https://github.com/platformio

### CODE REVIEW

File: etc/inc/allow-common-devel.inc [hunk 1]
```diff
@@ -38,3 +38,10 @@ noblacklist ${HOME}/.bundle
 # Rust
 noblacklist ${HOME}/.cargo
 noblacklist ${HOME}/.rustup
+
+#Arduino
```
Comment (kmk3) [inline etc/inc/allow-common-devel.inc:42]:
```suggestion
# Arduino
```

File: etc/inc/allow-common-devel.inc [hunk 2]
```diff
@@ -38,3 +38,10 @@ noblacklist ${HOME}/.bundle
...
+noblacklist ${HOME}/.arduino15
+noblacklist ${HOME}/Arduino
+
+# PlatformIO (can be used with Arduino)
+noblacklist ${HOME}/.platformio
```
Comment (kmk3) [inline etc/inc/allow-common-devel.inc:47]: This is not blacklisted anywhere; add it to disable-programs.inc.

File: etc/inc/allow-common-devel.inc (hunk 1 above)
Comment (glitsj16) [inline etc/inc/allow-common-devel.inc:42]: Code looks fine. We do `sort` these groups `alphabetically`. Please move it accordingly.

File: etc/inc/allow-common-devel.inc [hunk 3]
```diff
@@ -38,3 +38,10 @@ noblacklist ${HOME}/.bundle
 # Rust
 noblacklist ${HOME}/.cargo
 noblacklist ${HOME}/.rustup
+
+#Arduino
+noblacklist ${HOME}/.arduino15
+noblacklist ${HOME}/Arduino
+
+# PlatformIO (can be used with Arduino)
```
Comment (glitsj16) [inline etc/inc/allow-common-devel.inc:46]: Code looks fine. We do `sort` these groups `alphabetically`. Please move it accordingly.

**Review by kmk3 — CHANGES_REQUESTED** [review]

**Review by glitsj16 — CHANGES_REQUESTED** [review]
Keep entree blocks sorted alphabetically. Otherwise a clean PR :+1:.

File: etc/inc/allow-common-devel.inc (hunk 1 above)
Comment (marek22k) [reply etc/inc/allow-common-devel.inc]: Thanks for the hint!

**Review by kmk3 — APPROVED** [review]

**Review by glitsj16 — APPROVED** [review]
LGTM

**Pull request merged**